task5/task2.py

```diff
@@ -0,0 +1,103 @@
+"""Task5."""
+
+import argparse
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument("--year", help="displays Top250 movies titles with year")
```

Comment:
if you define arguments in this way I should print `python main.py --year whatever` to print titles with year. You can use `action="store_true"` option instead. 
It also concerns --rate and --all keys.
You can find examples in presentation for 5th lesson